<ui version="4.0" >
 <class>measurepulseClass</class>
 <widget class="QDialog" name="measurepulseClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:15pt; font-weight:600;">摩擦系数测试&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>第一次</string>
   </property>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>长度（mm）</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>140</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>第二次</string>
   </property>
   <widget class="QPushButton" name="pushButton_4" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>长度（mm）</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>210</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>第三次</string>
   </property>
   <widget class="QPushButton" name="pushButton_6" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>230</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>长度（mm）</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_7" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>320</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>320</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
